pages/api/v1/bounty/getBountyInfoFromIndex.ts

```
@@ -0,0 +1,79 @@
+import { LISTING_LIMIT } from '~src/global/listingLimit';
+
+interface Args {
+	bountyIndex: number;
+	network: string;
+}
+
+export async function getBountyInfo({ bountyIndex, network }: Args): Promise<IApiResponse<{ status: string; reqAmount?: string; curator?: string; createdAt?: string }>> {
+	try {
+		if (!network || !isValidNetwork(network)) throw apiErrorWithStatusCode(messages.INVALID_NETWORK, 400);
+
+		if (isNaN(bountyIndex)) throw apiErrorWithStatusCode(messages.INVALID_PARAMS, 400);
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)